packages/table/src/table.tsx

```diff
@@ -51,6 +53,14 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     fillBodyWithGhostCells?: boolean;
 
     /**
+     * If exists, a callback that returns the data for a specific cell. This need not
+     * match the value displayed in the `<Cell>` component. The value will be
+     * invisibly added as `textContent` into the DOM before copying. If not exists,
+     * copy via hotkeys (mod+c) will not work.
```

Comment:
For example, @adidahiya, if they select a whole column, we'd want to copy all cells in the columns, but due to virtualized rendering, there's no guarantee that all the cells in that column will have ever been rendered. Furthermore, some cells have truncated content (like JSON content), so copying the actual cell text would be incomplete.